integreat_cms/cms/linklists.py

```diff
@@ -11,6 +13,21 @@ class PageTranslationLinklist(Linklist):
     model = PageTranslation
     html_fields = ["content"]
 
+    @classmethod
+    def filter_callable(cls, objects):
+        """
+        Get only latest versions for same page and language
+
+        :param objects: Objects to be filtered
+        :type objects: ~django.db.models.query.QuerySet
+        :return: Objects that passed the filter
+        :rtype: ~django.db.models.query.QuerySet
+        """
+        latest = PageTranslation.objects.filter(
+            page_id=OuterRef("page_id"), language_id=OuterRef("language_id")
+        ).order_by("-version")
+        return objects.filter(version=Subquery(latest.values("version")[:1]))
```

Comment:
```suggestion
        return objects.distinct("page", "language")
```
I think since the translations are already ordered by the version, a `distinct` is a bit simpler and should do the same in this case.

Reply:
Thank you for your advice, you are right, this query is much simpler :slightly_smiling_face: